guidelines/index.html

```diff
@@ -1,352 +1,320 @@
+                  </dd><dd>
+                    <a href="https://github.com/w3c/wcag/commits/">コミット履歴</a>
+                  </dd>
         
-        <a href="https://www.w3.org/"><abbr title="World Wide Web Consortium">W3C</abbr></a><sup>®</sup>
-        (<a href="https://www.csail.mit.edu/"><abbr title="Massachusetts Institute of Technology">MIT</abbr></a>,
-        <a href="https://www.ercim.eu/"><abbr title="European Research Consortium for Informatics and Mathematics">ERCIM</abbr></a>,
-        <a href="https://www.keio.ac.jp/">Keio</a>, <a href="http://ev.buaa.edu.cn/">Beihang</a>).
+        <dt>実装報告書:</dt><dd>
+                <a href="https://www.w3.org/WAI/WCAG21/implementation-report/">https://www.w3.org/WAI/WCAG21/implementation-report/</a>
+              </dd>
         
-        <abbr title="World Wide Web Consortium">W3C</abbr> <a href="https://www.w3.org/Consortium/Legal/ipr-notice#Legal_Disclaimer">liability</a>,
-        <a href="https://www.w3.org/Consortium/Legal/ipr-notice#W3C_Trademarks">trademark</a> and
-        <a rel="license" href="https://www.w3.org/Consortium/Legal/copyright-documents">document use</a>
-        rules apply.
-      
-      </p>
-  <hr title="Separator for header">
+        
+        <dt>前の勧告:</dt><dd><a href="https://www.w3.org/TR/WCAG20/">https://www.w3.org/TR/WCAG20/</a></dd>
+        <dt>編集者:</dt><dd class="editor p-author h-card vcard" data-editor-id="39770">
+    <a class="ed_mailto u-email email p-name" href="mailto:akirkpat@adobe.com">Andrew Kirkpatrick</a> (<span class="p-org org h-org">Adobe</span>)
+  </dd><dd class="editor p-author h-card vcard" data-editor-id="41218">
+    <a class="ed_mailto u-email email p-name" href="mailto:josh@interaccess.ie">Joshue O Connor</a> (<span class="p-org org h-org">Invited Expert, InterAccess</span>)
+  </dd><dd class="editor p-author h-card vcard" data-editor-id="44689">
+    <a class="ed_mailto u-email email p-name" href="mailto:acampbell@nomensa.com">Alastair Campbell</a> (<span class="p-org org h-org">Nomensa</span>)
+  </dd><dd class="editor p-author h-card vcard" data-editor-id="34017">
+    <a class="ed_mailto u-email email p-name" href="mailto:cooper@w3.org">Michael Cooper</a> (<span class="p-org org h-org">W3C</span>)
+  </dd>
+        
+        
+        <dt>フィードバック:</dt><dd>
+        <a href="https://github.com/w3c/wcag/">GitHub w3c/wcag</a>
+        (<a href="https://github.com/w3c/wcag/pulls/">pull requests</a>,
+        <a href="https://github.com/w3c/wcag/issues/new/choose">new issue</a>,
+        <a href="https://github.com/w3c/wcag/issues/">open issues</a>)
+      </dd><dd><a href="mailto:public-agwg-comments@w3.org?subject=%5BWCAG21%5D%20YOUR%20TOPIC%20HERE">public-agwg-comments@w3.org</a> <kbd>[WCAG21] <em>… メッセージトピック …</em></kbd>の件名にて (<a rel="discussion" href="https://lists.w3.org/Archives/Public/public-agwg-comments">アーカイブ</a>)</dd>
```

Comment:
少なくとも以下のはずですが、これ翻訳する必要ありますかね…
```suggestion
      </dd><dd><a href="mailto:public-agwg-comments@w3.org?subject=%5BWCAG21%5D%20YOUR%20TOPIC%20HERE">public-agwg-comments@w3.org</a> <kbd>[WCAG21] <em>… メッセージトピック …</em></kbd>の件名にて (<a rel="discussion" href="https://lists.w3.org/Archives/Public/public-agwg-comments">archives</a>)</dd>
```